calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+# marker_radar_lidar_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/marker_radar_lidar_calibrator.md)
+
+## Purpose
+
+The package `marker_radar_lidar_calibrator` allows extrinsic calibration between radar and 3d lidar sensors used in autonomous driving and robotics.
+
+Currently, the calibrator only supports the radar that includes distance and azimuth angle, but without elevation angle. For example, ARS 408 radar can be calibrated with this tool. Also, note that the 3d lidar should have a resolution that is high enough to scan several points on the radar reflector (calibration target).
+
+## Inner-workings / Algorithms
+
+The calibrator is designed to accurately predict the transformation between radar and lidar sensors. It starts by pinpointing the central points of reflectors within lidar pointclouds and radar messages, then aligns these points for precise matching. An SVD-based and a yaw-only rotation estimation algorithm are applied to these correlated points to determine the transformation. Specifically, the calibration process consists of four primary steps: constructing a background model, extracting the foreground to detect reflectors, matching and filtering lidar and radar detections, and finally executing the calibration.
```

Comment:
We do not predict.
What are the basis of saying "accurately"? It is better not to use this terms unless there are numbers to back it up

Reply:
In several parts, your paragraphs are just too long. Try to break them down to make them easier to read

Reply:
"and radar messages, then aligns these points "
The current use of the connector here seems wrong to me

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/ef622282783c63cc215288d1cd2f015f45aa840c and https://github.com/tier4/CalibrationTools/pull/164/commits/57f8ddad3e9c5421ca59c90df572b4ba27dc74d7

Reply:
messages is changed to object, and the points is regarding to the central points.

Reply:
I will repeat it again. We do not predict

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/4872fda383f963dd042fac4324780ef6848eb2af